driver/src/main/scala/com/datastax/spark/connector/mapper/GettableDataToMappedTypeConverter.scala

```diff
@@ -175,7 +175,7 @@ class GettableDataToMappedTypeConverter[T : TypeTag : ColumnMapper](
       for ((s, _) <- columnMap.setters)
         yield (s, ReflectionUtil.methodParamTypes(targetType, s).head)
     val setterColumnTypes: Map[String, ColumnType[_]] =
-      columnMap.setters.mapValues(columnType)
+      columnMap.setters.map{case (k, v) => (k, columnType(v))}.toMap
```

Comment:
Just curious, why did you choose `.map().toMap` over `.view.mapValues...`?

Reply:
IIRC I was struggling to find a syntax that works in both `2.12 and `2.13`. It looks like `mapValues` was actually fine as long as the return type is converted back to a map.
[[SPARKC-686] Replace closure with idiomatic mapValues](https://github.com/datastax/spark-cassandra-connector/pull/1361/commits/1e5702e3acd4d5f5f01c5c3d8cd8e090a7c052a7)